src/pykiso/lib/auxiliaries/udsaux/common/uds_callback.py

```diff
@@ -294,15 +296,17 @@ def handle_data_download(
                 if data[0] != expected_pci:
                     block_data_len += len(data) - 1
                     log.internal_warning(
-                        f"Consecutive frame missed: "
-                        f"expected PCI {hex(expected_pci)}, got {hex(data[0])}"
+                        "Consecutive frame missed: expected PCI "
+                        f"{hex(expected_pci)}, got {hex(data[0])}",
```

Comment:
stil f string ? was this intended ?

Reply:
Yes, Sebastian Clerson said : 
It makes less sense to have percent formatting for warning and error levels as these are usually never filtered out

And since there is an if condition you do not render the f string everytime but only when there is an error